README.md

```diff
@@ -56,14 +56,26 @@ Adds the following methods to the redux store
 
 - `boolean` - Is the IPFS instance ready to use yet?
 
+#### `store.selectIpfsInitFailed()`
+
+- `boolean` - Did the IPFS instance failed to start?
```

Comment:
Did the IPFS instance failed to start?
-> 
Did the IPFS instance fail to start?